boards/arm/ti_am62x_sk_m4/ti_am62x_sk_m4_defconfig

```diff
@@ -0,0 +1,33 @@
+# Texas Instruments Sitara AM62x-SK-M4 EVM
+#
+# Copyright (c) 2023 Texas Instruments Incorporated
+# Copyright (c) 2023 L Lakshmanan
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Platform Configuration
+CONFIG_SOC_SERIES_AM62X_M4=y
+CONFIG_SOC_AM62x_M4=y
+CONFIG_BOARD_TI_AM62X_SK_M4=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_SEMIHOST=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+
+# Enable RAT module
+CONFIG_EXTERNAL_ADDRESS_TRANSLATION=y
```

Comment:
Could these be the defaults for the soc series?
Same for `CONFIG_UART_NS16550_ACCESS_WORD_ONLY`

Reply:
Will add them to the defaults!